cmd/pipecd/Dockerfile

```diff
@@ -1,3 +1,26 @@
+# web builder
+FROM node:20.18.1-alpine3.21 AS web
+
+WORKDIR /app
+
+COPY . .
+
+RUN apk add --no-cache make
+RUN make update/web-deps
+RUN make build/web
+
+# pipecd builder
+FROM golang:1.23.3 AS builder
+
+WORKDIR /app
+
+COPY go.* ./
+RUN go mod download
+
+COPY . ./
+
+RUN make build/go MOD=pipecd
+
```

Comment:
Sorry, we use `git` in our Makefile in many places, so please install git in web and go builder?
https://github.com/pipe-cd/pipecd/blob/d5ba3d2949bd1dc794020f36fafafb61ca749e44/Makefile#L21

Reply:
@Warashi Thank you for your review. I installed `git` command to builder container on below commit.
https://github.com/pipe-cd/pipecd/pull/5420/commits/beb90e71020a14194b606a6f4d66d913c61a09b8

 And `golang:1.23.3` has been installed `git` already.
```console
$ docker run --rm -it golang:1.23.3 git --help
usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>] [--config-env=<name>=<envvar>]
           <command> [<args>]

These are common Git commands used in various situations:

start a working area (see also: git help tutorial)
   clone     Clone a repository into a new directory
   init      Create an empty Git repository or reinitialize an existing one

work on the current change (see also: git help everyday)
   add       Add file contents to the index
   mv        Move or rename a file, a directory, or a symlink
   restore   Restore working tree files
   rm        Remove files from the working tree and from the index

examine the history and state (see also: git help revisions)
   bisect    Use binary search to find the commit that introduced a bug
   diff      Show changes between commits, commit and working tree, etc
   grep      Print lines matching a pattern
   log       Show commit logs
   show      Show various types of objects
   status    Show the working tree status

grow, mark and tweak your common history
   branch    List, create, or delete branches
   commit    Record changes to the repository
   merge     Join two or more development histories together
   rebase    Reapply commits on top of another base tip
   reset     Reset current HEAD to the specified state
   switch    Switch branches
   tag       Create, list, delete or verify a tag object signed with GPG

collaborate (see also: git help workflows)
   fetch     Download objects and refs from another repository
   pull      Fetch from and integrate with another repository or a local branch
   push      Update remote refs along with associated objects

'git help -a' and 'git help -g' list available subcommands and some
concept guides. See 'git help <command>' or 'git help <concept>'
to read about a specific subcommand or concept.
See 'git help git' for an overview of the system.
```